database/migrations/create_entries_table_with_string_ids.php.stub

```diff
@@ -10,22 +10,18 @@ return new class extends Migration {
         Schema::create($this->prefix('entries'), function (Blueprint $table) {
             $table->uuid('id');
             $table->string('site')->index();
-            $table->uuid('origin_id')->nullable();
```

Comment:
Was thing change from `uuid` to `unsignedBigInteger` intentional? The origin id would also be a uuid.